lib/matobj.gi

```diff
@@ -1579,15 +1579,16 @@ InstallMethod( \[\,\]\:\=, "for a matrix object, two positions, and an object",
 ##  <#GAPDoc Label="MultMatrixRow">
 ##  <ManSection>
 ##  <Oper Name="MultMatrixRow" Arg='mat,i,elm' Label=""/>
+##  <Oper Name="MultMatrixRowLeft" Arg='mat,i,elm' Label=""/>
 ##  
 ##  <Returns>nothing</Returns>
 ##
 ##  <Description>
 ##  <P/>
-##  For a matrix <A>mat</A> such that <A>mat</A> is mutable, a row index <A>i</A>  
-##  and a scalar <A>elm</A> this operation multiplies <A>elm</A> with the <A>i</A>-th row 
-##  in-place.
-##  Note that by default <A>elm</A> is multiplied from the left. 
+##  Multiplies the <A>i</A>-th row of the mutable matrix <A>mat</A> with the scalar 
+##  <A>elm</A> from the left in-place.
+##  <P/>
+##  <C>MultMatrixRowLeft</C> is a synonym for <C>MultMatrixRow</C>.
```

Comment:
The recommended GAPDoc syntax for references of documented variables would look as follows.
`<Ref Oper="MultMatrixRowLeft"/> is a synonym of <Ref Oper="MultMatrixRow"/>.`

Reply:
Also inside the definition of `MultMatrixRow` and `MultMatrixRowLeft` itself?

Reply:
Yes, GAPDoc is smart enough to detect this.

Reply:
I tried replacing this line. Those are the warnings I get:
`#W WARNING: non resolved reference: rec(
  Oper := "MultMatrixRowLeft" )
#W WARNING: non resolved reference: rec(
  Oper := "MultMatrixRow" )`

Reply:
Cross-references via `<Ref>` elements work only if the referenced element can be matched; if it has a label then the `<Ref>` element needs this label, too. From another point of view: Labels make sense only to distinguish several `<Oper>` or so elements with the same identifier, and then a `<Ref>` element can point to a specific one, via the label.

Reply:
Ok, thanks for the clarification!